package.json

```diff
@@ -56,6 +56,7 @@
     "mongodb": "^3.7.3",
     "mongoose": "^5.13.14",
     "mongoose-validator": "^2.1.0",
+    "morgan": "^1.10.0",
```

Comment:
Could you explain why we need `morgan` library in this PR?

Reply:
I am sorry I was using this library  localy to debug the application

Reply:
Done, Removed!